docs/data/hubble/analyst-guide/queries-for-horizon-like-data.mdx

```diff
@@ -0,0 +1,922 @@
+
+Learn more about [accounts](../../../learn/glossary.mdx#account).
+
+### General account information and XLM balance
+
+```sql
+select
+  account_id
+  , sequence_number as sequence
+  , sequence_ledger
+  , sequence_time
+  , num_subentries
+  , home_domain
+  , last_modified_ledger
+  , balance
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.accounts`
+where true
+  -- equivalent account_id parameter
+  and account_id=<account_id>
+  -- filter to the specific date or date range
+  and batch_run_date between '2024-01-01' and '2024-01-02'
+order by closed_at desc
+```
+
+### Non-XLM asset balance information
+
+```sql
+select
+  account_id
+  , asset_code
+  , asset_issuer
+  , asset_type
+  , balance
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.trust_lines`
+where true
+  -- equivalent account_id parameter
+  and account_id=<account_id>
+  -- filter to the specific date or date range
+  and batch_run_date between '2024-01-01' and '2024-01-02'
+order by closed_at desc
+```
+
+### Account signer information
+
+```sql
+select
+  account_id
+  , signer
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.account_signers`
+where true
+  -- equivalent account_id parameter
+  and account_id=<account_id>
+  -- filter to the specific date or date range
+  and batch_run_date between '2024-01-01' and '2024-01-02'
+order by closed_at desc
+```
+
+### List all accounts
+
+```sql
+select
+  account_id
+from `crypto-stellar.crypto_stellar_dbt.accounts_current`
+where true
+-- limit does not reduce the amount of data BigQuery queries
+limit 1000
+```
+
+### Retrieve an Account’s Transactions
+
+```sql
+select
+  op_account_id
+  , transaction_id
+  , transaction_hash
+  , ledger_sequence
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
+where true
+  -- equivalent account_id parameter
+  and op_account_id=<account_id>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at >= timestamp(current_date)
+-- limit does not reduce the amount of data BigQuery queries
+limit 1000
+```
+
+### Retrieve an Account’s Operations
+
+```sql
+select
+  op_account_id
+  , op_id
+  , type_string
+  , ledger_sequence
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
+where true
+  -- equivalent account_id parameter
+  and op_account_id=<account_id>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at >= timestamp(current_date)
+-- limit does not reduce the amount of data BigQuery queries
+limit 1000
+```
+
+### Retrieve an Account’s Payments
```

Comment:
Does the Horizon logic for Payments only return direct payments? Path payments aren't included?

Reply:
I'm not sure. I'll ask the team

Reply:
Oops it's more than just direct payments
https://stellarfoundation.slack.com/archives/C02B04RMK/p1719950212626139